### PR TITLE
chore(package): Update to React 15.0.2

### PR DESCRIPTION
**Pull request by etaoins**
- Update our React dependency from 15.0.1 to 15.0.2

- Declare we want React 15 in our bower config. This allows us to install beside packages also expecting React 15

### CODE REVIEW

**Comment by davepoon** [comment]
LGTM :thumbsup:

File: bower.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "main": "dist/alexandria-main.js",
   "dependencies": {
     "lodash": "^3.10.1",
-    "react": "^0.14.7",
```
Comment (antklim) [inline bower.json]: From 0.14.7 to 15.0.0?

**Comment by antklim** [comment]
LGTM

**Comment by etaoins** [comment]
@davepoon @antklim Merge?

**Pull request merged**